bp_be/src/include/bp_be_internal_if_defines.vh

```diff
@@ -124,6 +124,7 @@
   /* TODO: make opcode */                                                                          \
   typedef struct packed                                                                            \
   {                                                                                                \
+    logic [dword_width_p-1:0]       cause;                                                         \
```

Comment:
Is this just used for validating against dromajo? (no hardware use?)

Reply:
Yes